packages/dev/core/src/Physics/physicsHelper.ts

```diff
@@ -810,6 +872,9 @@ class PhysicsVortexEvent {
      * @param force
      */
     public dispose(force: boolean = true) {
+        if (!this._cylinder) {

```

Comment:
I wonder if dispose should also set _cylinder to null or undefined in this case ?

Reply:
Fixed